MapboxAndroidDemo/src/main/AndroidManifest.xml

```diff
@@ -183,8 +183,6 @@
                 android:value="com.mapbox.mapboxandroiddemo.MainActivity" />
         </activity>
 
-        <service android:name=".account.AccountRetrievalService" />
```

Comment:
We need this service for signing into a Mapbox account. Please add it back into the manifest file.

Reply:
I'll do it